driver-bindings/src/wrappers.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::bindings::ne_enclave_start_info;
+
+impl Default for ne_enclave_start_info {
+    fn default() -> Self {
+        Self {
+            flags: 0,
+            enclave_cid: 0,
+        }
+    }
+}
```

Comment:
nit: May be also move here the aliases from `resource_manager.rs` if you are going to move it to separate repo later?
```
/// CamelCase alias for the bindgen generated driver struct (ne_enclave_start_info).
pub type EnclaveStartInfo = ne_enclave_start_info;

/// CamelCase alias for the bindgen generated driver struct (ne_user_memory_region).
pub type UserMemoryRegion = ne_user_memory_region;

/// CamelCase alias for the bindgen generate struct (ne_image_load_info).
pub type ImageLoadInfo = ne_image_load_info;
```

Reply:
I thought about this a lot myself. It's a really good pattern to provide sane wrappers over plain FFI bindings (e.g. https://github.com/rust-vmm/kvm-bindings/blob/main/src/x86/fam_wrappers.rs), and my first attempt was to move these inside the bindings crate, but then I realized we don't add any APIs to them whatsoever. More so, we use them as C structs (a few lines below the snippet you showed we size_of() on one of these structs). In their current state, these aliases wouldn't provide any real value if they were apart of the bindings crate.

If you're ok with it, we can move them if/when their API evolves beyond their current impl.

Reply:
Yep, I'm ok with that, my only idea was to at least bring naming style closer to Rust. But it's fine for now.